<p>A <code>throw</code> statement exit from a method and so on and so on or it is caught by a <code>try</code>/<code>catch</code> block. It does not return a variable but an exception:</p>